### DISCUSSION TITLE
Export/snapshot management in the backoffice

### DISCUSSION THREAD

**Discussion by FransdeJong** [umbraco/Umbraco.Deploy.Issues#312] [Ideas]
A question I get a lot from my customers is the abiltity to make a content snapshot before large changes and the ability to restore them when needed.

This is a broad complex request and I don't know of a super solution but I can see the potential of this request.
We could use this route to upgrade to a new version of Umbraco in a different environment for example.

So a export site and import and maybe even a partial import would be awesome.

**Comment by ronaldbarendse** [comment]
Hi Frans! Part of this feature request is now available within Deploy: you can do exports before making large changes and import them again. I'll leave this open as a feature/idea discussion, because we do want to look into having a dashboard that shows all exports (snapshots) and allows users to download or import them easily...
